test/OpenTelemetry.Instrumentation.Http.Tests/OpenTelemetry.Instrumentation.Http.Tests.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.1" Condition="'$(TargetFramework)' != 'net452'" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.1"/>
```

Comment:
Curious, I saw that we have different versions of this package used in different places:
1. is this intentional (to have multiple versions used)?
2. do we want to extract the version to the common place?

Reply:
http tests had no need of this package. it was accidently part of it. removed